### ISSUE TITLE
Non-java Modules don't get tested

### ISSUE THREAD

**Issue by kpagratis** [dropbox/AffectedModuleDetector#235]
I have a repo with a module that uses node. It uses the base and node gradle plugins and registers a list of tasks, `test` being among them.

When running `./gradlew runAffectedUnitTests -Paffected_module_detector.enable` the affected module is found but it's never listed in the list of modules to decide whether to execute against.

For example, the bottom of the output is like so 
```
[INFO] [amd] dependents of :node-module is [:node-module]
[INFO] [amd] checking whether I should include :a-module and my answer is false
[INFO] [amd] checking whether I should include :another-module and my answer is false
```

`node-module` is not listed and the tests are never run. It seems like the detector only operates against java/kotlin modules. Is this by design?

**Comment by joshafeinberg** [comment]
Correct, we check if you have either the java, kotlin, or android plugins on that module

https://github.com/dropbox/AffectedModuleDetector/blob/main/affectedmoduledetector/src/main/kotlin/com/dropbox/affectedmoduledetector/AffectedModuleDetectorPlugin.kt#L278
